HesoyamHospital/GraphicEditor/Service/DrawingShapesService.cs

```diff
@@ -125,16 +127,25 @@ public void MouseRightButtonDown(object sender, MouseButtonEventArgs e)
             if (rectangle.Name.Contains("room"))
             { 
                 Information information = new Information();
-                information.name.Text = rectangle.Name;
-                information.visiting.Text = Global.AdditionalInformation.VisitingHours;
-                information.working.Text = Global.AdditionalInformation.WorkingHours;
-                information.doctor.Text = Global.AdditionalInformation.Doctor;
+                RoomService roomService = Backend.AppResources.getInstance().roomService;
+                Room room = roomService.GetRoomByName(rectangle.Name);
+
+                information.name.Text = room.RoomNumber;
+                if (room.Occupied)
```

Comment:
This part can be replaced with ternary operator in order to reduce number of code lines.